pkg/controller/jenkins/reconciler.go

```diff
@@ -57,11 +55,10 @@ const (
 type JenkinsReconciler struct {
 	// This client, initialized using mgr.Client() above, is a split client
 	// that reads objects from the cache and writes to the apiserver
-	client    client.Client
-	dynClient dynamic.Interface
-	scheme    *runtime.Scheme
-	logger    logr.Logger
-	result    reconcile.Result
+	client client.Client
+	scheme *runtime.Scheme
+	logger logr.Logger
+	result reconcile.Result
```

Comment:
Can we add errors over here namely
```
errors utils.Errors
```
This would allow for the errors to be more localized to Jenkins Reconciler.